lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java

```diff
@@ -672,12 +672,14 @@ public void shouldThrowOnInvalidIssuedAtIfPresent() throws Exception {
     @Test
     public void shouldOverrideAcceptIssuedAtWhenIgnoreIssuedAtFlagPassedAndSkipTheVerification() throws Exception {
         Clock clock = mock(Clock.class);
-        when(clock.getToday()).thenReturn(new Date(DATE_TOKEN_MS_VALUE - 1000));
+        when(clock.getToday()).thenReturn(new Date(DATE_TOKEN_MS_VALUE - 10000));
```

Comment:
The purpose of the test I understand it, is to verify that if using a custom issuedAt leeway that would cause verification to fail, when calling `ignoreIssuedAt` the verification would still pass. If setting the mock clock to return one second earlier (subtracting 1000), the verification would still pass if not using `ignoreIssuedAt`, unless setting the leeway to zero. That means the test would pass even if `ignoredIssuedAt` were not actually ignoring the issuedAt validation. For context:

- JWT issuedAt date:  Sat Jan 17 20:26:32 CST 1970
- Setting mock clock to `DEFAULT_TOKEN_MS_VALUE - 1000)`:   Sat Jan 17 20:26:31 CST 1970 (1 second before issuedAt date)
- Setting mock clock to` DEFAULT_TOKEN_MS_VALUE - 10000)`:  Sat Jan 17 20:26:22 CST 1970 (10 seconds before issuedAt date)